modules/virtualisation/containers.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.virtualisation.containers;
+
+  inherit (lib) mkOption types;
+
+  toml = pkgs.formats.toml { };
+in {
+  meta.maintainers = [ pkgs.lib.maintainers.michaelCTS ];
+  options.virtualisation.containers = {
+
+    enable = lib.mkEnableOption "the common containers configuration module";
+
+    ociSeccompBpfHook.enable = lib.mkEnableOption "the OCI seccomp BPF hook";
+
+    # Unimplemented as podman modifies the file and it would have to be merged
+    #  not replaced when activating a new config
+    #    containersConf.settings = mkOption {
+    #      type = toml.type;
+    #      default = { };
+    #      description = "The {file}`containers.conf` configuration.";
+    #    };
```

Comment:
Present in `nixpkgs` https://github.com/NixOS/nixpkgs/blob/230262cda5dc5b704aae91273e2676a9ac65c81e/nixos/modules/virtualisation/containers.nix#L31-L35


Podman modifies the files in the user's home, outside of `home-manager`. Fixing that in this PR is out of scope, so not sure if this comment (and related commented out code) should stay or just be removed.

Reply:
I'll give that a try and see if podman complains :)